### DISCUSSION TITLE
.mqh .mq5 .mq4 files converted to Binary Files

### DISCUSSION THREAD

**Discussion by mananslush** [community/community#31164] [Other Feature Feedback, Questions, & Ideas]
Whenever I upload .mqh .mq4 .mq5 files on Github, it says "Binary file not shown."
These are actual code files, can Github implement a suitable solution for these?

**Comment by jfullstackdev** [comment]
maybe report it here by creating a PR, just make sure you follow their format

https://github.com/github/linguist
